MIGRATION.md

```diff
@@ -2851,7 +2851,7 @@ npm install core-js@^3.0.1 --save-dev
 
 ### Args passed as first argument to story
 
-Starting in 6.0, the first argument to a story function is an [Args object](https://storybook.js.org/docs/react/api/csf#args-story-inputs). In 5.3 and earlier, the first argument was a [StoryContext](https://github.com/storybookjs/storybook/blob/next/lib/addons/src/types.ts#L49-L61), and that context is now passed as the second argument by default.
+Starting in 6.0, the first argument to a story function is an [Args object](https://storybook.js.org/docs/react/api/csf#args-story-inputs). In 5.3 and earlier, the first argument was a [StoryContext](https://github.com/storybookjs/storybook/blob/next/code/lib/addons/src/types.ts#L49-L61), and that context is now passed as the second argument by default.
```

Comment:
@joeycozza can you check this link for accuracy, I was checking this and this, and this doesn't seem right. The `code/lib/addons/src` directory doesn't provide a `types.ts` file, as you can see [here](https://github.com/storybookjs/storybook/tree/next/code/lib/addons/src).

Reply:
Good catch. 
Since that is talking about "In 5.3 and earlier..." I decided to point to it when it existed back in 5.3

https://github.com/storybookjs/storybook/blob/release/5.3/lib/addons/src/types.ts#L24-L31